### DISCUSSION TITLE
Details / Summary: Accessibility concerns

### DISCUSSION THREAD

**Discussion by shawnthompson** [gc-da11yn/gc-da11yn.github.io#176] [Best Practices (Open Discussions)]
When using the `details` elements, to use the `open="open"` by default.

This will allow users to use the browsers find feature and not exclude any text found inside the `details` element.

This will also allow users who use an immersive reader to read the text and not have to interrupt their reading to expand the `summary` and break their concentration. 

This is has been a concern in the W3C Cognitive task force with the W3.org website.

**Comment by delisma** [comment]
But what if the content inside a detail/summary add little value to the overall content?

**Reply by shawnthompson** [reply]
Why put it in a `details` to hide it in first view?

Also something in a collapsed `details` element can not be found using the browser's find feature.

**Reply by delisma** [reply]
I would ask Lisa Fast about her take on it. I know that we're putting long descriptions of images in details/summary. I check what other recommended use cases there is.

**Reply by shawnthompson** [reply]
My main concern is find-ability when using `details`. Ask users to find topics on a page that the content is completely in `details` elements.

Like the [Standard on Web Accessibility- Canada.ca](https://www.tbs-sct.canada.ca/pol/doc-eng.aspx?id=23601).
